2-mandatory/1-writers.js

```diff
@@ -82,10 +88,12 @@ Exercise 3:
 
 function logAliveWritersInTheirForties() {
   // write your code here
+    if((w.age>= 40 && w.age<= 49)&& w.alive == true){
```

Comment:
Again, the loop is missing

Can you think of shorter way to write `w.alive === true`?